content/sensu-go/5.8/sensuctl/reference.md

```diff
@@ -320,6 +320,15 @@ sensuctl check info check-cpu --format wrapped-json
 In addition to the standard operations, commands may support subcommands or flags that allow you to take special action based on the resource type; the following sections call out those resource-specific operations.
 For a list of subcommands specific to a resource, run `sensuctl TYPE --help`.
 
+### Handling large datasets
+
+When querying sensuctl for large datasets, you use the `--chunk-size` flag with any `list` command to avoid timeouts and improve performance.
```

Comment:
```suggestion
When querying sensuctl for large datasets, you can use the `--chunk-size` flag with any `list` command to avoid timeouts and improve performance.
```